test/symbol/Network.spec.ts

```diff
@@ -28,7 +28,10 @@ describe('Symbol Network', () => {
     it('can create correct hasher for Symbol', () => {
         const network = new SymbolNetwork('public', 0x68, '57F7DA205008026C776CB6AED843393F04CD458E0AA2D9F1D5F31A402072B2D6');
         const hasher = network.addressHasher();
-        expect(hasher instanceof SHA3).to.be.true;
+        const padding = Object.entries(hasher.create())[2][1];
+        const sha3_256Padding = Object.entries(sha3_256.create())[2][1];
+
+        expect(padding).to.be.deep.equal(sha3_256Padding);
```

Comment:
would it be more natural to test that a known value (e.g. 'a' is hashed correctly); then this test is independent of hash lib used.